build.gradle

```diff
@@ -69,6 +69,9 @@ allprojects {
                 driver
                 utilities
             }
+    configurations.all {
+        exclude group: "log4j", module:"log4j"
+    }
```

Comment:
Just to make sure I'm understanding: unlike the prior iterations/commits this will block loading of log4j (instead of loading a dummy version) and fail at the module with some kind of Not Found error?

Rather than loading the insecure version.

Reply:
Correct.  If there is indeed a dependency on log4j, there will likely be a "ClassNotFound" error of some sort. Though the cause of the build error may not be as immediately obvious, this seems preferable to me because (a) it's much less of a hack (b) it achieves what we actually want, so we won't fail a build if there's a transitive dependency that doesn't actually get used (c) it's a once-and-for-all kind of fix.